langgraph/demo.py

```
@@ -0,0 +1,151 @@
+# db
+from sqlalchemy import create_engine, text
+import asyncio
+def directory_db_query(query_string):
+    print("executing db query for directory db: " + query_string)
+    results = []
+    try:
+        engine = create_engine('postgresql+psycopg2://admin:example@localhost:5432/directory')
+        connection = engine.connect()
+        my_query = text(query_string)
+        results = connection.execute(my_query).fetchall()
+    except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)